sites/public/src/components/listing/ListingViewSeeds.tsx

```diff
@@ -0,0 +1,752 @@
+                </div>
+              )}
+              {(listing.depositMin || listing.depositMax) && (
+                <div className={styles["split-card-cell"]}>
+                  <Heading size={"md"} className={styles["thin-heading"]}>
+                    {t("t.deposit")}
+                  </Heading>
+                  <div className={styles.emphasized}>
+                    {getCurrencyRange(parseInt(listing.depositMin), parseInt(listing.depositMax))}
+                  </div>
+                  <div>{listing.depositHelperText}</div>
+                </div>
+              )}
+            </div>
+            {listing.costsNotIncluded && (
+              <div className={"seeds-m-be-content"}>{listing.costsNotIncluded}</div>
+            )}
+            {enableUtilitiesIncluded && utilitiesIncluded?.length && (
+              <div className={"seeds-m-be-content"}>
+                <Heading size={"md"}>{t("listings.sections.utilities")}</Heading>
+                {utilitiesIncluded}
+              </div>
+            )}
+          </Card.Section>
+        </Card>
+      )}
+    </>
+  )
+
+  const FeaturesSection = (
+    <CollapsibleSection
+      title={t("listings.sections.featuresTitle")}
+      subtitle={t("listings.sections.featuresSubtitle")}
+      priority={2}
+    >
+      <div className={`${styles["mobile-inline-collapse-padding"]} seeds-m-bs-section`}>
+        {features.map((feature, index) => {
+          return (
+            <HeadingGroup
+              heading={feature.heading}
+              subheading={feature.subheading}
+              size={"lg"}
+              headingPriority={3}
+              className={`${styles["heading-group"]} seeds-m-be-content`}
+              key={index}
+            />
+          )
+        })}
+        <Heading size={"lg"} className={"seeds-m-be-header"} priority={3}>
+          {t("t.unitFeatures")}
+        </Heading>
+        {listing?.unitsSummarized?.byUnitType.map((summary, index) => {
+          const unitTableData = getUnitTableData(listing.units, summary)
+          return (
+            <div className={index !== 0 ? "seeds-m-bs-header" : ""} key={index}>
+              <ExpandableSection
+                title={unitTableData.barContent}
+                priority={4}
+                disableCollapse={listing.disableUnitsAccordion}
+                uniqueId={`unit-feature-${index}`}
+              >
+                <StandardTable headers={unitsHeaders} data={unitTableData.unitsFormatted} />
+              </ExpandableSection>
+            </div>
+          )
+        })}
+        {AdditionalFees}
+      </div>
+    </CollapsibleSection>
+  )
+
+  const NeighborhoodSection = (
+    <CollapsibleSection
+      title={t("t.neighborhood")}
+      subtitle={t("listings.sections.neighborhoodSubtitle")}
+      priority={2}
+    >
+      <div className={`${styles["mobile-inline-collapse-padding"]} seeds-m-bs-section`}>
+        <ListingMap
```

Comment:
The design has a "get directions" link below the map. Which looks like new functionality
![image](https://github.com/user-attachments/assets/5a9ec02a-cc7c-4a0a-9e55-e6163a20067c)

Reply:
I missed this thank you!